devel/playdate-sdk/Portfile

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                playdate-sdk
+version             1.12.2
+revision            0
+categories          devel
+maintainers         @idleberg
+license             Restrictive
+
+description         Playdate SDK
+long_description    This package installs includes a rich set of APIs, a \
+                    compiler, and documentation for Playdate game development.
+
+homepage            https://play.date/dev/
+master_sites        https://download-keycdn.panic.com/playdate_sdk/
+distname            PlaydateSDK-${version}
+distfiles           ${distname}.zip
```

Comment:
if this uses a ZIP file, you probably just should set `use_zip yes` and then the `distfiles` field and manually `unzip` in the extract phase shouldn't be required